### ISSUE TITLE
[enhancement] Add initial preset.

### ISSUE THREAD

**Issue by iojforever** [SebasCapo/Lights#7]
**Initial preset idea:**

For now, there was an initial delay added into the plugin, so i think it would be cool if there will be an initial preset. 

**WDYM?**

As i saw, there is only two ways to launch presets - random one and one-after-another. With that scheme, i would like to make one preset that will be first one and after that, others will go through the round. Also, if possible, that preset should not be included into main presets list.

**Thats not really hard i guess?**
Well, if its not, then how about make multiple initial presets possible with configurable time for every preset? Just an idea for you to look at.

**Lest pretend its possible. What about an example?**
Sure!

    # Initial preset that will be the first and only one. 
    - Preset3
    # The order on which presets will be ran.
    order:
    - Preset1
    - Preset2
    - Preset4
    - Preset5

**Comment by ImBeryl** [comment]
Good idea! Will include this change next patch, ideally this will be added without breaking pre-4.0.2 files, so it won't be strictly necessary to use it.
